framework/Application.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, arg__1, messageHandler, interactionType=Interaction.Yes):
       self.printTag = 'RAVEN Application'
       self.UIs = {}
       self.interactionType = interactionType
-      qtg.QApplication.__init__(self,arg__1)
+      qtw.QApplication.__init__(self,arg__1)
```

Comment:
I was using the inherited argument name from [QApplication](https://srinikom.github.io/pyside-docs/PySide/QtGui/QApplication.html#PySide.QtGui.QApplication). I can change it to `arg1` in order to comply with RAVEN standards, so long as it does not lose its semantic meaning from the Qt side.